guardian/core.py

```diff
@@ -157,9 +157,11 @@ def get_perms(self, obj):
             if guardian_settings.AUTO_PREFETCH:
                 return []
             if self.user and self.user.is_superuser:
-                perms = list(chain(*Permission.objects
-                                   .filter(content_type=ctype)
-                                   .values_list("codename")))
+                perms = list(
+                    chain.from_iterable(
+                        Permission.objects.filter(content_type=ctype).values_list("codename")
+                    )
+                )
```

Comment:
The simpler solution is to pass `flat=True` to `values_list`.
```suggestion
                perms = list(
                    Permission.objects.filter(content_type=ctype).values_list("codename", flat=True)
                )
```